conda_smithy/feedstock_content/ci_support/disabled.svg

```diff
@@ -0,0 +1,20 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="92" height="20">
```

Comment:
I would probably put this somewhere else (`conda_smithy/static/disabled.svg` or something), but otherwise, 👍 .